### ISSUE TITLE
RSS feed times are effected by timezone of builder

### ISSUE THREAD

**Issue by aschmahmann** [ipfs/distributions#330]
Despite all the times being published as UTC two different builds result in different times. For example,

```
<       <pubDate>Wed, 06 Mar 2019 23:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 07 Mar 2019 05:00:00 GMT</pubDate>
47c50
<       <pubDate>Sun, 07 Oct 2018 22:00:00 GMT</pubDate>
---
>       <pubDate>Mon, 08 Oct 2018 04:00:00 GMT</pubDate>
55c58
<       <pubDate>Wed, 13 Jan 2021 23:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 14 Jan 2021 05:00:00 GMT</pubDate>
63c66
<       <pubDate>Wed, 13 Jan 2021 23:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 14 Jan 2021 05:00:00 GMT</pubDate>
71c74
<       <pubDate>Wed, 13 Jan 2021 23:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 14 Jan 2021 05:00:00 GMT</pubDate>
79c82
<       <pubDate>Wed, 30 Sep 2020 22:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 01 Oct 2020 04:00:00 GMT</pubDate>
87c90
<       <pubDate>Sun, 10 Dec 2017 23:00:00 GMT</pubDate>
---
>       <pubDate>Mon, 11 Dec 2017 05:00:00 GMT</pubDate>
95c98
<       <pubDate>Mon, 10 Oct 2016 22:00:00 GMT</pubDate>
---
>       <pubDate>Tue, 11 Oct 2016 04:00:00 GMT</pubDate>
103c106
<       <pubDate>Wed, 22 Apr 2020 22:00:00 GMT</pubDate>
---
>       <pubDate>Thu, 23 Apr 2020 04:00:00 GMT</pubDate>
111c114
<       <pubDate>Mon, 11 May 2020 22:00:00 GMT</pubDate>
---
>       <pubDate>Tue, 12 May 2020 04:00:00 GMT</pubDate>
```

Which is conveniently the timezone difference between me and the previous publisher.

**Comment by jessicaschilling** [comment]
@aschmahmann Do you have the bandwidth to PR a fix for this?

**Comment by aschmahmann** [comment]
Not sure what my schedule looks like next week yet. If it seems easy enough I can try a PR fix before the go-ipfs v0.8.0 release.

**Comment by lidel** [comment]
_(writing this in case someone ever finds this useful in the future)_

The gist of the problem is that `date` in [dist.json](https://dist.ipfs.io/go-ipfs/v0.9.0/dist.json) manifests is test like `June 23, 2021` and does not include timezone and does not use ISO format.

`Date` from JS uses local timezone:

> ![2021-07-30--03-17-08](https://user-images.githubusercontent.com/157609/127585385-d66c6200-6acb-4101-8a00-a6f3256e11cf.png)

### Quick fix for now

I did a quick fix in  **TODO**  by appending UTC before parsing the string:

> ![2021-07-30--03-19-41](https://user-images.githubusercontent.com/157609/127585569-0e78773c-85e7-46c0-8c16-f91df52ff939.png)

Usually this is discouraged due to inconsistent behavior across browsers, but here we only care about behavior in Nodejs during the build.


### Alternative fixes

An alternative is to switch new `dist.json` to use military format, which defaults to UTC:

> ![2021-07-30--03-36-39](https://user-images.githubusercontent.com/157609/127586794-6b18684d-d037-4364-b077-2e5f739c0be2.png)

or a  deterministic ISO format:

> ![2021-07-30--03-41-51](https://user-images.githubusercontent.com/157609/127587155-25f335a4-3700-4dac-acc1-8e4ee5237d17.png)

Decided to not  do that, as it requires changing historical manifests and/or may break scripts that expect current format.
